lib/partitions_validator_utils.pm

```diff
@@ -55,9 +56,10 @@ sub validate_filesystem {
 }
 
 sub validate_read_write {
-    my $args = shift;
-    assert_script_run("echo Hello > $args->{mount_point}/emptyfile", fail_message => 'Failure while writing in ' . $args->{mount_point});
-    assert_script_run("grep Hello $args->{mount_point}/emptyfile",   fail_message => 'Failure while reading from ' . $args->{mount_point});
+    my $args      = shift;
+    my $emptyfile = catfile($args->{mount_point}, 'emptyfile');
```

Comment:
it would be more clear to call it $path or $emptyfilepath for the reader so it would not be needed to check documentation and see if it is really getting the content or returning the path.

Reply:
no problem. it kept the used name as it was tho. no change on it. i will address it

Reply:
fixed